CMakeLists.txt

```diff
@@ -7,12 +7,21 @@ cmake_minimum_required(VERSION 2.6)
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -std=gnu99 -fstack-protector-all")
 if(CMAKE_BUILD_TYPE STREQUAL Debug)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -Wall -Wextra -Wformat-security -Wmissing-prototypes -Wmissing-declarations -Wdeclaration-after-statement -Winit-self -Waggregate-return -Wmissing-format-attribute -Wundef -Wbad-function-cast -Wwrite-strings -Wformat=2")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -Wall -Wextra -Wformat-security -Winit-self -Wundef -Wwrite-strings -Wformat=2 -Werror=missing-prototypes -Werror=missing-declarations -Wmissing-format-attribute -Werror=implicit-function-declaration -Werror=bad-function-cast -Werror=return-type")
```

Comment:
that's a very long line and I heard cmake only recently supports line breaking with `\`, however could be rewritten/split into multiple `set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -W<something> -W<somemore> ...` statements to stay within reasonable numbers of char per line (<120?) and make it more readable

Reply:
done